### ISSUE TITLE
urbit-king: Expected @t, but got ^

### ISSUE THREAD

**Issue by juaningan** [urbit/urbit#2721]
I'm getting this error after personal breach in urbit-king daemon, then exit:

`2020-04-11 14:52:30.069800: [error] ("Somthing died","compute thread",BadPleaNoun ("work", 171600, "...") ["Plea","%work","Work","(,)","2","Jammed","(,,)","3","BlipEv","%ames","AmesEv","%crud","(,,)","2","Cord","utf8-atom"] "Expected @t, but got ^")
`
Urbit binary works well. Except for the message reported in #2682 .

**Comment by belisarius222** [comment]
@joemfb I feel like this is probably related to your recent %crud/$goof rework.

**Comment by joemfb** [comment]
That's right, the structure of the `%crud` error notification events changes in #2366, from:

```
[%crud term tang]
```

to:

```
[%curd goof ovum]
```

As a reminder

```
|%
+$  card  [term *]
+$  goof  [term tang]
+$  ovum  [wire card]
+$  tang  (list tank)
+$  term  @tas
+$  wire  path
```

In other words, we wrap the crash report (bail mote and stack trace) around the original event (card thereof), and then poke arvo with that (on the original wire).

**Comment by joemfb** [comment]
Both old and new style `%crud` events are currently supported, so, properly speaking the type is actually:

```
[%crud $^([goof curd] [term tang])]
```

**Comment by joemfb** [comment]
Fixed in #2820.
